### DISCUSSION TITLE
安装v1.10.14-lts后出现一些问题

### DISCUSSION THREAD

**Discussion by BG6HJE-edu** [1Panel-dev/1Panel#6320] [Q&A]
大家好，今天下午在研究给面板绑定域名，先前失败几次后均通过 `1pctl reset domain` 成功重置，期间更新了29号新发布的 `v1.10.14-lts` 版本，而后便无法重置，具体表现为执行 `1pctl reset domain` 后访问 `http://ip:port` 仍然被重定向到先前设置的域名（域名绑定仍未成功，所以重定向后无法访问面板）。于是我执行 `1pctl uninstall` 和 `curl -sSL https://resource.fit2cloud.com/1panel/package/quick_start.sh -o quick_start.sh && sudo bash quick_start.sh` （即官网给出的一键安装脚本命令）试图手动回滚面板版本来解决问题，但是安装完成后正确输入账密无法登录面板，具体表现为弹出“登陆成功”消息后自动回到登录界面并需要重新输入密码。求教大家如何破局。

**Comment by ssongliu** [comment]
用无痕模式试试能否正常打开

**Reply by BG6HJE-edu** [reply]
你好，感谢回复。目前面板可以正常打开了，但是不能管理之前安装好的OpenRestry和MySQL，提示未安装，请问这种情况有没有解决办法？
